ale_linters/c/clangtidy.vim

```diff
@@ -0,0 +1,107 @@
+" Author: vdeurzen <tim@kompiler.org>, w0rp <devw0rp@gmail.com>,
+" gagbo <gagbobada@gmail.com>, Andrej Radovic <r.andrej@gmail.com>
+" Description: clang-tidy linter for c files
+
+call ale#Set('c_clangtidy_executable', 'clang-tidy')
+" Set this option to check the checks clang-tidy will apply.
+" The number of checks that can be applied to C files is limited in contrast to
+" C++
+"
+" Here is an incomplete list of C-compatible checks for clang-tidy version 6:
+" bugprone-suspicious-memset-usage
```

Comment:
Sure. I put the list here because the documentation only lists all of the checks, without any indication if they work for C, or just C++. I actually went through all of them by hand and picked out those that work for C.